### ISSUE TITLE
Fix S4275 FP when method is used inside getter or setter

### ISSUE THREAD

**Issue by andrei-epure-sonarsource** [SonarSource/sonar-dotnet#2435]
### Description

When the field is returned or set using a method call, the rule raises an issue.

The problem is that we don't really support inter-procedural analysis, so this FP is non-trivial to fix.

### Repro steps

```
    public class CrossProcedural
    {
        private int myValue;
        public int MyValue
        {
            // below there are two False Positives
            get => this.GetMyValue(); // Noncompliant {{Refactor this getter so that it actually refers to the field 'myValue'.}}
            set => this.SetMyValue(value); // Noncompliant {{Refactor this setter so that it actually refers to the field 'myValue'.}}
        }

        private int GetMyValue() => this.myValue;
        private void SetMyValue(int value) => this.myValue = value;
    }
```

### Related information

* SonarC# Version 7.14
